### ISSUE TITLE
Panel is invoked without having focus

### ISSUE THREAD

**Issue by humdingerb** [HaikuArchives/ClipUp#4]
Apparently the panel doesn't have focus when invoked either from the Deskbar icon nor via SHIFT+ALT+C. Therefore, keyboard navigation doesn't work, because ClipUp doesn't receive the key events.
